src/d3a/d3a_core/sim_results/area_throughput_stats.py

```diff
@@ -83,20 +88,26 @@ def update_results(self, area):
                 self.update_results(child)
 
     def aggregate_exported_imported_energy(self, area):
+        if area.uuid not in self.imported_energy:
+            self.imported_energy[area.uuid] = {}
+        if area.uuid not in self.exported_energy:
+            self.exported_energy[area.uuid] = {}
+
         past_markets = list(area._markets.past_markets.values())
         if len(past_markets) > 0:
             current_market = past_markets[-1]
         else:
             return
+
         child_names = [area_name_from_area_or_iaa_name(c.name) for c in area.children]
         for trade in current_market.trades:
             if child_buys_from_area(trade, area.name, child_names):
-                add_or_create_key(self.exported_energy, current_market.time_slot_str,
+                add_or_create_key(self.exported_energy[area.uuid], current_market.time_slot_str,
                                   trade.offer.energy)
             if area_sells_to_child(trade, area.name, child_names):
-                add_or_create_key(self.imported_energy, current_market.time_slot_str,
+                add_or_create_key(self.imported_energy[area.uuid], current_market.time_slot_str,
                                   trade.offer.energy)
-        if current_market.time_slot_str not in self.imported_energy:
-            self.imported_energy[current_market.time_slot_str] = 0.
-        if current_market.time_slot_str not in self.exported_energy:
-            self.exported_energy[current_market.time_slot_str] = 0.
+        if current_market.time_slot_str not in self.imported_energy[area.uuid]:
+            self.imported_energy[area.uuid][current_market.time_slot_str] = 0.
+        if current_market.time_slot_str not in self.exported_energy[area.uuid]:
+            self.exported_energy[area.uuid][current_market.time_slot_str] = 0.
```

Comment:
Ah yes, this is a remnant of moving this out of the AreaStats into AreaThroughputStats. Thanks for fixing!